CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
-## 1.1.1-dev
-
+## 1.2.0-dev
+
+* Support branch level coverage information, when running tests in the Dart VM.
+  This is not supported for web tests yet.
+* Add flag `--branch-coverage` (abbr `-b`) to collect_coverage that collects
+  branch coverage information. The VM must also be run with the
+  `--branch-coverage` flag.
+* Add flag `--pretty-print-branch` to format_coverage that works
+  similarly to pretty print, but outputs branch level coverage, rather than
+  line level.
+* Update `--lcov` (abbr `-l`) in format_coverage to output branch level
```

Comment:
Is this a breaking change?

Reply:
I don't think so. I'm adding significant functionality, but not changing existing functionality, so I think that makes it a minor version bump.

Reply:
Does any of our internal testing infrastructure consume `lcov` reports? Will they be broken with the change in format? If not, then it's probably fine to model as a non breaking releases.

Reply:
The lcov output was also modified when I added function coverage, and that didn't seem to be an issue, so I guess not.